src/query.js

```diff
@@ -111,8 +111,8 @@ export async function parquetQuery(options) {
 /**
  * Reads a list rows from a parquet file, reading only the row groups that contain the rows.
  * Returns a sparse array of rows.
- * @import {ParquetQueryFilter, ParquetReadOptions} from '../src/types.d.ts'
- * @param {ParquetReadOptions & { rows: number[] }} options
+ * @import {ParquetQueryFilter} from '../src/types.d.ts'
+ * @param {BaseParquetReadOptions & { rows: number[] }} options
```

Comment:
It's another breaking change, potentially more complex than `parquetReadObjects`, because here we are forbidding `rowFormat: "array"`, but we might imagine that (despite the erroneous return type `Promise<Record<string, any>[]>`), users might want to use this function with the `array` row format